modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -143,7 +143,7 @@ I: To watch your cluster installation logs, run 'rosa logs install -c <cluster_n
 <1> When creating the cluster, the listed `OpenShift version` options include the major, minor, and patch versions, for example `4.8.9`.
 <2> If more than one matching set of account-wide roles are available in your account for a cluster version, an interactive list of options is provided.
 <3> Multiple availability zones are recommended for production workloads. The default is a single availability zone.
-<4> Enable this option if you are using your own AWS KMS key to encrypt the control plane data volumes and the PVs for your applications. Specify the ARN for the KMS key that you added the account-wide role ARN to in the preceding step.
+<4> Enable this option if you are using your own AWS KMS key to encrypt the control plane data volumes for your applications. Specify the ARN for the KMS key that you added the account-wide role ARN to in the preceding step.
```

Comment:
You could remove "for your applications", because control plane data volumes relate to core OpenShift data.